### PR TITLE
Add the ability to cancel an analysis

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by FredKSchott** [comment]
Closing this PR as abandoned.

@rictic please take a look and rebase/reopen if this is something you'd still like to get merged in.

**Comment by rictic** [comment]
bump

File: src/core/cancel-token.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {CancelToken} from 'cancel-token';
+
+/**
+ * The analyzer only uses one method from CancelTokens, and it's likely to be
+ * the least contentious, so even if other parts change on the way to
+ * standardization,
```
Comment (justinfagnani) [inline src/core/cancel-token.ts]: standardization? :)

Comment (rictic) [reply src/core/cancel-token.ts]: Weird looking, but I think that's the right spelling: standardization

**Pull request merged**